### PR TITLE
build(FEC-10700): Improvement for CI/CD

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

1. add the number of commits from the master - added automatically by standard-version. (need static prefix and sha isn't needed anymore).
2. handle two calls after deployment for each repo.
3. releases from the no-master branch won’t be taken to beta/latest release.
4. curl request check failure.
5. change dependencies to work against the master for testing

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Comment by dan-ziv** [comment]
@Yuvalke so the change is actually to update UI conf when tagging on master branch but to avoid it when tagging on side branch?

**Comment by Yuvalke** [comment]
> @Yuvalke so the change is actually to update UI conf when tagging on master branch but to avoid it when tagging on side branch?

Yes, there are also cases for failure that I didn't cover before

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-flow": "^7.10.4",
     "@babel/register": "^7.10.5",
-    "@playkit-js/playkit-js": "0.63.0",
+    "@playkit-js/playkit-js": "canary",
```
Comment (OrenMe) [inline package.json:41]: can we make this explicitly done on the canary job only? add a step to install canary

Comment (Yuvalke) [reply package.json:41]: it's only for tests

**Review by OrenMe — APPROVED** [review]

**Review by OrenMe — APPROVED** [review]

**Pull request merged**